ci/build_docs.sh

```diff
@@ -56,4 +63,5 @@ popd
 
 RAPIDS_VERSION_NUMBER="${RAPIDS_VERSION_MAJOR_MINOR}" rapids-upload-docs
 
+rapids-logger "Test script exiting with latest error code: $EXITCODE"
```

Comment:
It's not the "latest" -- this would exit 1 if an earlier command exited 1, even if the most recent command exited 0.
```suggestion
rapids-logger "Test script exiting with exit code: $EXITCODE"
```